remote/new.go

```diff
@@ -175,3 +219,21 @@ func NewV1Image(baseImageRepoName string, keychain authn.Keychain, ops ...func(*
 	options.Platform = processPlatformOption(options.Platform)
 	return processImageOption(baseImageRepoName, keychain, options.Platform, options.RegistrySettings)
 }
+
+// ValidateRepoName
+// TODO move this code to something more generic
+func validateRepoName(repoName string, o *imgutil.IndexOptions) error {
```

Comment:
This function will be more re-usable if it just accepts an insecure bool, instead of index options. Then we can use it for images too.

Reply:
I agreed, honestly I put it here just to remember me to do something with it later!

Reply:
Just a note, we're not actually validating the "repo name" for remote __images__ today (though maybe we should). I think I removed it in https://github.com/husni-faiz/imgutil/pull/12, but we could add it back.